Centaurus.Test.Utils/GlobalInitHelper.cs

```diff
@@ -0,0 +1,75 @@
+﻿using Centaurus.Domain;
+using Centaurus.Models;
+using stellar_dotnet_sdk;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Centaurus.Test
+{
+    public static class GlobalInitHelper
+    {
+        public static void Setup(string[] clientSecrets, string[] auditorSecrets, BaseSettings settings)
```

Comment:
It would be nice to see a few comments and arguments description for all methods here. Developers need to understand what this init helper does and which arguments are required for different cases.